### ISSUE TITLE
for not working as expected with AsyncIterable?

### ISSUE THREAD

**Issue by stereosteve** [oscarotero/vento#28]
Had a `dir.vto` file in a lume 2 application like this:

```
{{Array.from(Deno.readDirSync(".")).map(f => f.name).join(",")}}

{{ for file of Array.from(Deno.readDirSync(".")) }}
<div> 1: {{file.name}}</div>
{{ /for }}

{{ for file of Deno.readDirSync(".") }}
<div> 2: {{file.name}}</div>
{{ /for }}

{{ for await file of Deno.readDir(".") }}
<div> 3: {{file.name}}</div>
{{ /for }}
```

Only the `1` prefix version shows up... `2` and `3` both missing.

Based on the [docs](https://vento.js.org/syntax/for/#async-iterators) I would have thought 2 and 3 would work...

**Comment by oscarotero** [comment]
Hi!
Thanks for bringing this up.
I just released the version 0.10.2 with support for iterables.
Feel free to reopen the issue if you find any issue.

**Comment by stereosteve** [comment]
Funny.  I took a stab at it [here](https://github.com/oscarotero/vento/compare/main...stereosteve:vento:iterable).

Pretty similar.

Thanks for the quick fix!
